### PR TITLE
ci: only install yq with snap.

### PR DESCRIPTION
**Pull request by ioga**
## Description

Don't install `yq` twice.

## Checklist

- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](../docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1854,7 +1854,6 @@ jobs:
           executor: <<pipeline.parameters.machine-image>>
           extras-requires: anchorecli
       - run: sudo snap install yq
```
Comment (mackrorysd) [inline .circleci/config.yml:1856]: jq is definitely used in the script. I'd suggest changing line 1856 to `sudo snap install yq && sudo snap install jq`?

**Review by mackrorysd — CHANGES_REQUESTED** [review]

File: .circleci/config.yml [hunk 2]
```diff
@@ -1853,8 +1853,7 @@ jobs:
       - setup-python-venv:
           executor: <<pipeline.parameters.machine-image>>
           extras-requires: anchorecli
-      - run: sudo snap install yq
-      - run: sudo apt-get install -y jq
+      - run: sudo snap install yq jq
```
Comment (mackrorysd) [inline .circleci/config.yml:1856]:
```suggestion
      - run: sudo snap install yq && sudo snap install jq
```

Comment (mackrorysd) [reply .circleci/config.yml:1856]: At least locally for me, you can't pass multiple dependencies to snap and install them at once.

**Review by mackrorysd — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 2 above)
Comment (ioga) [reply .circleci/config.yml:1856]: strange: I tried on circleci with ssh job and it worked. oh well.

**Review by mackrorysd — APPROVED** [review]

**Pull request merged**